sdk/appcenter-analytics/src/main/java/com/microsoft/appcenter/analytics/Analytics.java

```diff
@@ -877,6 +912,26 @@ private void setDefaultTransmissionTarget(String transmissionTargetToken) {
         }
     }
 
+    /**
+     * Set transmission interval. The interval should be between 3 seconds and 86400 seconds (1 day).
+     * Should be called before the service is started.
+     *
+     * @param seconds the latency of sending events to Analytics.
+     * @return <code>true</code> if the interval is set, <code>false</code> otherwise.
+     */
+    private boolean setInstanceTransmissionInterval(int seconds) {
+        if (mChannel != null) {
+            AppCenterLog.error(LOG_TAG, "Transmission interval should be set before the service is started.");
+            return false;
+        }
+        if (seconds < MINIMUM_TRANSMISSION_INTERVAL_IN_SECONDS || seconds > MAXIMUM_TRANSMISSION_INTERVAL_IN_SECONDS) {
+            AppCenterLog.error(LOG_TAG, "The transmission interval is not valid, it should be between 3 seconds and 1 day (86400 seconds).");
```

Comment:
Use constants instead of 3 seconds and 1 day in logs. There was a comment about that some time ago but I guess it was overwritten? https://github.com/microsoft/appcenter-sdk-android/pull/1145#discussion_r282121071